### DISCUSSION TITLE
欢迎来到 OLAINDEX 讨论区

### DISCUSSION THREAD

**Discussion by WangNingkai** [WangNingkai/OLAINDEX#346] [General]
## 👋 欢迎你!

我们正在使用 GitHub Deiscussion 作为一个地方与我们社区的其他成员联系。我们希望您

- 提出你想知道的问题。
 
- 分享想法。
 
- 与其他社区成员互动。
 
- 欢迎他人并保持开放的心态。请记住，这是一个我们的社区

共建💪

要想开始，请在下面留言，介绍自己，并告诉我们你在这个社区做什么。

**Comment by misaka123456** [comment]
![QQ截图20201231001819](https://user-images.githubusercontent.com/38593541/103366522-d96e5500-4afd-11eb-8658-3b41dfcd78a7.png)

我稍微改动了一下布局，这样是不是好看一点
